plugins/GSdx/GSTextureCache.h

```diff
@@ -104,11 +105,12 @@ class GSTextureCache
 		void Update();
 	};
 
+	using FastSourcePtrList = FastList<Source*, 4>;
```

Comment:
They are equivalent in this case, but if `using` is never used elsewhere in the code, I'll also use `typedef`.